osmaxx/excerptexport/urls.py

```diff
@@ -3,28 +3,24 @@
 from django.views.generic import TemplateView
 
 from osmaxx.excerptexport.views import (
-    list_downloads,
```

Comment:
Yes, I wanted to keep it until I've got a working replacement, which looks very much like it is the case :-)